mrun

```diff
@@ -1,3 +1,2 @@
 #! /bin/sh
-
-exec python "mesos_scheduler.py" --mpi "$@"
+exec python "$PARACEL_INSTALL_PREFIX/mesos_scheduler.py" --mpi "$@"
```

Comment:
Yes, this way came to my mind firstly. But the concern here is usage of `readlink` is very different from osx to linux.